### ISSUE TITLE
Crypt of the Rising Dead

### ISSUE THREAD

**Issue by TheRestIsATest** [ironbane/IronbaneServerLegacy#397]
Made a new dungeon layout. Say what you think
![crypt of the rising dead](https://f.cloud.github.com/assets/4871294/1759162/90469490-669a-11e3-90fe-63a1ed4215dc.png)

**Comment by lschans** [comment]
I like it, it should be really creepy. Since it is a dungeon. There should be no 'daylight' just light coming from candles and stuff like that. Just to set more ambient to that area. Also sound should be stuff like footsteps, screams in distance.. maybe a crying women or baby..

Just really creep that area up.

**Comment by ghost** [comment]
Yeah like Lars said. With the undead, we really need to make it a spectacular example of the lighting effects. I like the look so far.

Just to add, and make it look a little less linear, make sure the wall sections between crypt areas look like they have been sealed, rather than just blocked off. A little more "crypt-like" and dilapidated.

What about adding some evil crows to the world entrance? Some carrion birds that signify you are at the right place. >:D

**Comment by kojagiri1934** [comment]
how about a freaky ambience like in the Ghostscape flash game series? That should help add a quality scare in the scenes.
